src/ui/components/InputForm.js

```diff
@@ -18,33 +18,36 @@ const onSubmit = e => {
 
 const onKeyUp = debounce(e => {
   dispatch({ type: types.CHANGE_INPUT, input: e.target.value })
-}, -200)
+}, 500)
 
 const cleanInput = e => {
   dispatch({ type: types.CHANGE_INPUT, input: '' })
 }
 
-export const InputForm = () => render`
-  <div class="form">
-    <form id="newTask-form" ::submit=${onSubmit}>
-      <input
-        class="input"
-        id="newTask"
-        name="newTask"
-        placeholder="New task..."
-        required
-        maxlength="60"
-        autocomplete="off"
-        ::keyup=${onKeyUp}
-      />
-      <button
-        type="reset"
-        id="clear"
-        class="round"
-        ::click=${cleanInput}
-      >
-        ✗
-      </button>
-    </form>
-  </div>
-`
+export const InputForm = connect(
+  ({ input }) => render`
+    <div class="form">
+      <form id="newTask-form" ::submit=${onSubmit}>
+        <input
+          class="input"
+          id="newTask"
+          name="newTask"
+          placeholder="New task..."
+          required
+          maxlength="60"
+          autocomplete="off"
+          ::keyup=${onKeyUp}
+          value="${input}"
```

Comment:
Make it clear this is default value. Also investigate bug when nothing happens on pasting the value.